src/nvim/terminal.c

```diff
@@ -309,6 +309,14 @@ void terminal_resize(Terminal *term, uint16_t width, uint16_t height)
     }
```

Comment:
If I opened a terminal in one tab, and then opened the terminal buffer in another tab, it would seem that this will ignore the window in the other one. I think we should use `FOR_ALL_TAB_WINDOWS()` here, just ignoring the `tabpage` argument.